app/models/manageiq/providers/openstack/inventory/parser/common_methods.rb

```diff
@@ -0,0 +1,181 @@
+module ManageIQ::Providers::Openstack::Inventory::Parser::CommonMethods
```

Comment:
I decided to introduce this module to share some common code for cloud/infra. Not sure if this should be placed in parser.rb.